### PR TITLE
Added remember me function, hide categories on mobile

### PR DESCRIPTION
**Pull request by caike08**
As it is described itself, some features were added:
- Remember me now works properly
- Hide categories on mobile (provisory)

### CODE REVIEW

File: client/stylesheets/custom.less [hunk 1]
```diff
@@ -477,3 +477,9 @@ section {
     }
   }
 }
+
+#findBy {
+  @media(max-width: 480px) {
+    display: none;
+  }
+}
```
Comment (MarceloBorgesP) [inline client/stylesheets/custom.less:485]: try to always leave an extra line at the end @caike08

File: imports/ui/pages/home.js [hunk 2]
```diff
@@ -18,6 +24,12 @@ Template.home.events({
     //REMOVE ERRORS
     $('.warning').removeClass('warning');
 
+    console.log(event);
```
Comment (MarceloBorgesP) [inline imports/ui/pages/home.js:27]: And don't forget the logs

**Review by MarceloBorgesP — APPROVED** [review]
As our time is short I'm accepting, but we have to remember to fix this in future PRs

File: client/stylesheets/custom.less (hunk 1 above)
Comment (caike08) [reply client/stylesheets/custom.less:485]: Beautify problems, sorry!

**Pull request merged**